src/pages/auth/signup/Signup.jsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react'
+import { useState } from 'react'
+import axios from 'axios'
+import { useNavigate  } from 'react-router-dom'
+
+
 import Navbar from '../../../components/navbar/Navbar'
 
 const Signup = () => {
+  let navigate = useNavigate();
+  const [email , setemail ] = useState("onkarcool123@gmail.com")
+  const [password , setpassword ] = useState("OnkarDeshpande")
+  const [disabled , setdisabled] = useState(false)
+
+  const SignupHandler = () =>{
+    setdisabled(true)
+    const bodydata = {
+      "email":email,
+      "password":password,
+      "firstname":"Onkar",
+      "lastname":"Deshpande"
+    } 
+
+    axios.post('/api/auth/signup',bodydata)
+    .then((response)=>{
+      localStorage.setItem('token',response.data.encodedToken)
+      console.log('signed up successfully');
+      navigate('/Listvideos')
+    },
+    (error)=>{
+      console.log('error from signup');
```

Comment:
if possible...try using try..catch block to handle errors and exceptions

Reply:
Need to learn this. will implement this across projects when I do.